### ISSUE TITLE
'toil restart' doesn't work

### ISSUE THREAD

**Issue by joelarmstrong** [BD2KGenomics/toil#805]
Any `toil restart` will fail, as far as I can tell, because a "realTimeLogging" option needs to be created.

```
Traceback (most recent call last):
  File "/cluster/home/jcarmstr/venv/bin/toil", line 8, in <module>
    load_entry_point('toil==3.2.0a2', 'console_scripts', 'toil')()
  File "/cluster/home/jcarmstr/toil/src/toil/utils/toilMain.py", line 30, in main
    module.main()
  File "/cluster/home/jcarmstr/toil/src/toil/utils/toilRestart.py", line 59, in main
    toil.run()
  File "/cluster/home/jcarmstr/toil/src/toil/common.py", line 426, in run
    realtimeLogLevel = self.options.logLevel if self.options.realTimeLogging else None
AttributeError: 'Namespace' object has no attribute 'realTimeLogging'
```

I "fixed" this locally by adding a realTimeLogging option, but not sure if I should make a PR? There are some changes in #769 that affect the naming of some of the methods, but it still seems to expect realTimeLogging to be defined, so `toil restart` should still be broken in that PR as well.

**Comment by hannes-ucsc** [comment]
@JakeNarkizian Please try to reproduce this and then submit a PR with a minimal fix.

**Comment by hannes-ucsc** [comment]
@cket's PR is going to fix this.

**Comment by cket** [comment]
resolved by b9ae754

**Comment by hannes-ucsc** [comment]
https://github.com/BD2KGenomics/toil/commit/b9ae7542110026ea179ea876ca0ac48aec3d6742 obsoletes this.
